src/less/global/global.less

```diff
@@ -26,6 +26,12 @@ a {
     }
 
     &:hover {
-        .color-token(link-foreground-color-visited, color-foreground-visited);
+        opacity: 0.7;
+    }
+
+    &[disabled] {
```

Comment:
For links, we should support `&[aria-disabled="true"]` and `&:not([href])` over `&[disabled]`